src/Microsoft.TestPlatform.Build/Tasks/VSTestTask2.cs

```diff
@@ -40,6 +42,9 @@ public class VSTestTask2 : ToolTask, ITestTask
     public string? VSTestArtifactsProcessingMode { get; set; }
     public string? VSTestSessionCorrelationId { get; set; }
 
+    protected override Encoding StandardErrorEncoding => !_disableUtf8ConsoleEncoding ? Encoding.UTF8 : base.StandardErrorEncoding;
+    protected override Encoding StandardOutputEncoding => !_disableUtf8ConsoleEncoding ? Encoding.UTF8 : base.StandardOutputEncoding;
```

Comment:
Small nit for reading but we all have different tastes so feel free to ignore it

```suggestion
    protected override Encoding StandardErrorEncoding => _disableUtf8ConsoleEncoding ? base.StandardErrorEncoding ? Encoding.UTF8;
    protected override Encoding StandardOutputEncoding => _disableUtf8ConsoleEncoding ? base.StandardOutputEncoding : Encoding.UTF8;
```